### DISCUSSION TITLE
Convert string returned from function as bytes

### DISCUSSION THREAD

**Discussion by uhthomas** [cuelang/cue#839] [General] [answered]
Hi,

How can I make the following work?

```
import "encoding/yaml"

some_field: bytes

some_field: yaml.Marshal({})
```

Real world use-case: Kubernetes secrets.

**Comment by myitcv** [comment, marked as answer]
The canonical way to do this is via single-quote interpolation:

```
import "encoding/yaml"

some_field: bytes

some_field: '\(yaml.Marshal({}))'
```

**Reply by uhthomas** [reply]
Ah, perfect! Thank you!

**Comment by cueckoo** [comment]
This discussion has been migrated to https://github.com/cue-lang/cue/issues/839.

For more details about CUE's migration to a new home, please see https://github.com/cue-lang/cue/issues/1078.
